### PR TITLE
Added tags for Lyft/Motivate bikeshare systems

### PR DESCRIPTION
**Pull request by PierceWHoward**
I added tags to bikesharing systems owned and operated by Lyft/Motivate to make adding stations/docks easier.

### CODE REVIEW

**Review by arch0345 — CHANGES_REQUESTED** [review]
Thanks @PierceWHoward and welcome! I've left you some suggestions that should be reviewed before merging

File: data/brands/amenity/bicycle_rental.json [hunk 1]
```diff
@@ -2050,6 +2093,50 @@
...
+        "operator:type": "private",
+        "name": "Lyft Bike",
+        "fee": "yes",
+        "opening_hours": "24/7",
+        "phone": "+1-855-865-9553",
+        "website": "https://www.lyft.com/bikes/los-angeles-ca"
+      }
+    },
+    {
+      "displayName": "Lyft Bike",
```
[the remark this answers is not in the document]
Comment (PierceWHoward) [reply data/brands/amenity/bicycle_rental.json:2119]: Would it be okay if I add geojason files and use them?

**Comment by PierceWHoward** [comment]
I know that the new geojsons aren't that great but I wanted to make sure I included the entire area where locations were possible. Feel free to edit them in the future. :)

**Comment by arch0345** [comment]
Looks good! Could you also remove the website and phone tags?

**Comment by arch0345** [comment]
Since both Lyft Bikes have the same tags, I would just combine this into one entry that includes both the LA and CO geojsons in the locationSet

**Comment by PierceWHoward** [comment]
> Since both Lyft Bikes have the same tags, I would just combine this into one entry that includes both the LA and CO geojsons in the locationSet

Oops. My bad the reason I separated them initially was because of the different web addresses. Just pushed the change.

**Review by arch0345 — APPROVED** [review]

**Comment by PierceWHoward** [comment]
@arch0345 Should I remove opening_hours tags? Do you want me to make any additional changes?

**Comment by arch0345** [comment]
Don't worry, I removed it in the commit linked above. No other changes are needed - thanks for your first contribution!

**Pull request merged**